source/ambassador_host_test.go

```diff
@@ -141,3 +658,77 @@ func TestParseAmbLoadBalancerService(t *testing.T) {
 		}
 	}
 }
+
+func convertAmbassadorHostToUnstructured(hp *ambassador.Host, s *runtime.Scheme) (*unstructured.Unstructured, error) {
```

Comment:
I don't feel adding this to the global scope in this file is the right thing to do. There is already logic for converting using this https://github.com/kubernetes-sigs/external-dns/blob/master/source/ambassador_host.go#L121